### ISSUE TITLE
404 on the repo

### ISSUE THREAD

**Issue by dmaziuk** [zfsonlinux/zfs#3265]
Ermm... guys. I went through the motions as per #3250 and after
```
dkms remove --all zfs/0.6.3
 dkms remove --all spl/0.6.3
find /lib/modules/$(uname -r)/extra -name "splat.ko" -or -name "zcommon.ko" -or -name "zpios.ko" -or -name "spl.ko" -or -name "zavl.ko" -or -name "zfs.ko" -or -name "znvpair.ko" -or -name "zunicode.ko" | xargs rm -f
find /lib/modules/$(uname -r)/weak-updates/ -name "splat.ko" -or -name "zcommon.ko" -or -name "zpios.ko" -or -name "spl.ko" -or -name "zavl.ko" -or -name "zfs.ko" -or -name "znvpair.ko" -or -name "zunicode.ko" | xargs rm -f
yum reinstall zfs-release
```
I got
```
[root@hydra ~]# yum --enablerepo=zfs-testing reinstall zfs-dkms
...
zfs-dkms-0.6.3-133_g9635861.el FAILED
http://archive.zfsonlinux.org/epel-testing/7/x86_64/noarch/zfs-dkms-0.6.3-133_g9635861.el7.centos.noarch.rpm: [Errno 14] HTTP Error 404 - Not Found  ]  0.0 B/s |    0 B  --:--:-- ETA
Trying other mirror.

Error downloading packages:
  zfs-dkms-0.6.3-133_g9635861.el7.centos.noarch: [Errno 256] No more mirrors to try.
```
Same for spl-dkms. So now I have a bunch version control repositories sitting on a host with no zfs drivers other than those loaded in memory.

Can you **please** not post "solutions" that result in a complete clusteflop? "Stable to use in production" indeed.

Also #3258

**Comment by dmaziuk** [comment]
fixed: it's out as 0.6.4.

**Comment by dmaziuk** [comment]
PS. Thanks guys.
